packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -245,4 +253,35 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       allowMultiLogin: this.$config?.allowMultiLogin ?? true,
     };
   }
+
+  /**
+   * Called when we finish registration to silently handle the errors
+   */
+  private async defaultReLogin(): Promise<StationReLoginResponse | void> {
+    try {
+      const reLoginResponse = await this.services.agent.reload();
+
+      return reLoginResponse;
+    } catch (error) {
+      if (getErrorReason(error) === 'AGENT_NOT_FOUND') {
+        this.$webex.logger.info('Agent not found during re-login, handling silently');
+
+        return;
+      }
+      throw getErrorDetails(error, 'defaultReLogin');
+    }
+  }
+
+  /**
+   * Handle the custom event and call setAgentState for disconnect event
+   */
+  private async handleAgentWssDisconnect(msg: StateChange) {
```

Comment:
For handling the logic to set agent status to `AVAILABLE` again (edge case). Tested and it was working (in vidcast 1)